build/images/flow-aggregator/Dockerfile

```diff
@@ -7,14 +7,16 @@ COPY . /antrea
 
 # Make sure the flow-aggregator binary is statically linked.
 RUN CGO_ENABLED=0 make flow-aggregator
```

Comment:
If we use `ubuntu:20.04` as the base image, then we don't need to link statically any more.

Reply:
Done